otelcollector/test/containerstatus/container_status_test.go

```diff
@@ -20,12 +20,56 @@ var _ = DescribeTable("The containers should be running",
 	},
 	Entry("when checking the ama-metrics replica pod(s)", "kube-system", "rsName", "ama-metrics"),
 	Entry("when checking the ama-metrics-node", "kube-system", "dsName", "ama-metrics-node"),
+	Entry("when checking the ama-metrics replica pod(s)", "kube-system", "rsName", "ama-metrics", Label(utils.ARM64Label)),
+	Entry("when checking the ama-metrics-node", "kube-system", "dsName", "ama-metrics-node", Label(utils.ARM64Label)),
```

Comment:
The `Label(utils.ARM64Label)` is to tell ginkgo which tests to run when running the tests from the CLI. Lines 23-24 will run the same check as lines 21-22. A check for if the node the pod is on is arm64 for the daemonset pod would be needed

Reply:
Thanks! I have updated this in the last revision to check if pod is on arm64 node.